assets/src/blocks/Media/MediaFrontend.js

```diff
@@ -0,0 +1,51 @@
+import { MediaEmbedPreview } from './MediaEmbedPreview';
+import { MediaElementVideo } from './MediaElementVideo';
+
+const { __ } = wp.i18n;
+
+const wrapAndRemoveSize = embed_html => {
+  const wrapperDiv = document.createElement('div');
```

Comment:
Can't we do this before saving the embed html? Maybe here https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/376/files#diff-c6c91408622162ae2bd090b6d50b303d1de0a3ebff4e385997189643c4a18ce1R121

Reply:
Hm, initially I thought we could but I think we'll need to keep it in the MediaFrontend file for old content.

Reply:
If it's only for old content, we're already doing something for backwards compatibility in the back end, we might as well handle that there too. It only needs to remove the width and height attributes, and put it in a wrapper div. It's already doing the youtube url replacement https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/9fc1e628ca2a2bfc65d6570729afc9fac91793d2/classes/blocks/class-media.php#L124-L124

Reply:
Well, is not only for old content, is both for old and new content, maybe we should remove the PHP part instead.

Reply:
I went the other way around: removed it from PHP. Since it adds a wrapper div with a class, removes the width and height and replaces the URL, I thought it was more consistent to do it in a single place for all the cases. Using DOM methods is also a bit cleaner & safer than doing `str_replace` or `ereg_replace` in the PHP side.